src/Docker/Configuration/Container.php

```diff
@@ -35,6 +35,16 @@ public function getConfigTreeBuilder()
                             ->children()
                                 ->scalarNode('type')->end()
                                 ->scalarNode('context')->end()
+                                ->arrayNode('workspace_credentials')
+                                    ->ignoreExtraKeys(false)
+                                    ->children()
+                                        ->scalarNode('id')->isRequired()->end()
+                                        ->enumNode('type')->isRequired()
+                                            ->values(['snowflake'])
+                                        ->end()
+                                        ->scalarNode('#password')->end()
+                                    ->end()
+                                ->end()
```

Comment:
Myslim, ze to bude jeden z tech tasku, co sis uz zakladal, ale pro jistotu, tohle pridat i do https://github.com/keboola/job-queue/tree/main/libs/job-configuration

Reply:
Dík připsal jsem do https://keboola.atlassian.net/browse/KAB-313